### PR TITLE
Minor spelling/grammar fixes for READMEs.

### PR DESCRIPTION
**Pull request by davidshq**
Just a few minor improvements in these two READMEs.

### CODE REVIEW

**Comment by davidshq** [comment]
Added name to contributors to pass checks.

File: README.development [hunk 1]
```diff
@@ -114,18 +114,18 @@ User-contributed instructions for building using Cygwin:
         1. https://mlocati.github.io/articles/gettext-iconv-windows.html
```
Comment (evandrocoan) [inline README.development:114]: There is also `Advancded` three lines above this.

**Comment by davidshq** [comment]
@evandrocoan - Thanks! Fixed that and added to this pull request.

**Comment by dae** [comment]
Thanks David! Would you mind rebasing your changes to fix the conflict?

**Pull request closed without merging**